observability/monitoring/shiftstack-prometheus-configuration.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="shiftstack-prometheus-configuration"]
+= Monitoring clusters that run on OpenStack
+include::_attributes/common-attributes.adoc[]                   
+:context: shiftstack-prometheus-configuration
+                                                                
+toc::[]
+
+// TODO: Is this really RHOSO-first, and RHOSP will require tweaks? What's the audience impact if so?
+
+You can correlate observability metrics for clusters that run on {rh-openstack-first} or Red Hat OpenStack Services on OpenShift (RHOSO). By collecting metrics from both environments, you can monitor and troubleshoot issues across the infrastructure and application layers.
+
+There are two supported methods for metric correlation for clusters that run on {rh-openstack}:
+
+- https://prometheus.io/docs/practices/remote_write/#remote-write-tuning[Remote writing] to an external Prometheus instance.
+- Collecting data emitted from the federation endpoint of a cluster to a {rh-openstack}-hosted Prometheus instance.
```

Comment:
The target Prometheus instance is not just "a" Prometheus instance: it's really THE Prometheus instance of OpenStack that collects and exposes the OpenStack metrics. Do you think there can be a way to express this concept better? Please consider something like "to your {rh-openstack} observability stack"

@mandre ideas?

---

I'd also make it clearer that the Federation endpoint is OpenShift's.

Reply:
perhaps something like the following?
```suggestion
- Collecting data emitted from the {product-title} federation endpoint to the RHOSO observability stack.
```